src/main/plugins/currency-converter-plugin/currency-converter-plugin.ts

```diff
@@ -99,7 +100,7 @@ export class CurrencyConverterPlugin implements ExecutionPlugin {
     }
 
     private buildCurrencyConversion(userInput: string): CurrencyConversion {
-        const words = userInput.split(" ");
+        const words = userInput.trim().split(" ");
```

Comment:
Fixes an issue where the validator was checking against a trimmed input, but this was not
![image](https://user-images.githubusercontent.com/7577851/103457981-6d832b00-4d4f-11eb-90ac-12d29f0e7999.png)